### PR TITLE
Python compatibility and little fixes

### PR DESCRIPTION
**Pull request by jackokaiser**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (anandtrex) [inline LICENSE:1]: If you have no objection, I would prefer to go with GPLv3 instead.

File: setup.py [hunk 2]
```diff
@@ -1,17 +1,16 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(name='lsm',
       version='0.1',
       description='Liquid state machines (LSM) for spiking neural networks (SNN).',
-      #  url='...',  # TODO
+      url='https://github.com/HBPNeurorobotics/LSM',
```
Comment (anandtrex) [inline setup.py]: In this repository, it would make more sense for the url to be  'https://github.com/IGITUGraz/LSM'

**Review by anandtrex — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (jackokaiser) [reply LICENSE:1]: No problem - it's mostly to put a license in there, as the EU is quite picky with this

File: setup.py (hunk 2 above)
Comment (jackokaiser) [reply setup.py]: that's absolutely right - to be honest I thought the pull request was against the HBP Neurorobotics repo, but all good ;-)

**Pull request merged**